c-api/ResvgQt.h

```diff
@@ -441,15 +441,15 @@ class ResvgRenderer {
      */
     QRect viewBox() const
     {
-        return viewBoxF().toRect();
+        return viewBots().toRect();
```

Comment:
?!

Reply:
Thanks for catching this: was a slip in a case-insensitive search and replace.